<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qmycrap</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/app.png</normaloff>icons/app.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTreeView" name="treeView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>20</width>
        <height>0</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>RightToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionSave"/>
   <addaction name="actionSync"/>
   <addaction name="separator"/>
   <addaction name="actionImport"/>
   <addaction name="actionExport"/>
   <addaction name="separator"/>
   <addaction name="actionTop"/>
   <addaction name="actionBottom"/>
   <addaction name="separator"/>
   <addaction name="actionDate"/>
   <addaction name="separator"/>
   <addaction name="actionPreferences"/>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionRemove"/>
    <addaction name="separator"/>
    <addaction name="actionImport"/>
    <addaction name="actionExport"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionTop"/>
    <addaction name="actionBottom"/>
    <addaction name="separator"/>
    <addaction name="actionSync"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuInsert">
    <property name="title">
     <string>Insert</string>
    </property>
    <widget class="QMenu" name="menuRun">
     <property name="title">
      <string>Run</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icons/run.png</normaloff>icons/run.png</iconset>
     </property>
     <addaction name="actionFirstrun"/>
     <addaction name="actionRerun"/>
    </widget>
    <widget class="QMenu" name="menuPlatform">
     <property name="title">
      <string>Platform</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>icons/platform.png</normaloff>icons/platform.png</iconset>
     </property>
     <addaction name="actionPS1"/>
     <addaction name="actionPS2"/>
     <addaction name="actionPS3"/>
     <addaction name="actionPSP"/>
     <addaction name="actionPC"/>
     <addaction name="actionNES"/>
     <addaction name="actionSNES"/>
     <addaction name="actionGC"/>
     <addaction name="actionWII"/>
     <addaction name="actionGBA"/>
     <addaction name="actionNDS"/>
     <addaction name="actionGENESIS"/>
    </widget>
    <addaction name="actionDate"/>
    <addaction name="menuPlatform"/>
    <addaction name="menuRun"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuInsert"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionSave">
   <property name="icon">
    <iconset>
     <normaloff>icons/save.svg</normaloff>icons/save.svg</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset>
     <normaloff>icons/exit.svg</normaloff>icons/exit.svg</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionDate">
   <property name="icon">
    <iconset>
     <normaloff>icons/date.svg</normaloff>icons/date.svg</iconset>
   </property>
   <property name="text">
    <string>Date</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionFirstrun">
   <property name="icon">
    <iconset>
     <normaloff>icons/firstrun.png</normaloff>icons/firstrun.png</iconset>
   </property>
   <property name="text">
    <string>Firstrun</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionRerun">
   <property name="icon">
    <iconset>
     <normaloff>icons/rerun.png</normaloff>icons/rerun.png</iconset>
   </property>
   <property name="text">
    <string>Rerun</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionPS1">
   <property name="icon">
    <iconset>
     <normaloff>icons/ps1.png</normaloff>icons/ps1.png</iconset>
   </property>
   <property name="text">
    <string>PS1</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionPS2">
   <property name="icon">
    <iconset>
     <normaloff>icons/ps2.png</normaloff>icons/ps2.png</iconset>
   </property>
   <property name="text">
    <string>PS2</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionPS3">
   <property name="icon">
    <iconset>
     <normaloff>icons/ps3.png</normaloff>icons/ps3.png</iconset>
   </property>
   <property name="text">
    <string>PS3</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionPSP">
   <property name="icon">
    <iconset>
     <normaloff>icons/psp.png</normaloff>icons/psp.png</iconset>
   </property>
   <property name="text">
    <string>PSP</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionPC">
   <property name="icon">
    <iconset>
     <normaloff>icons/pc.png</normaloff>icons/pc.png</iconset>
   </property>
   <property name="text">
    <string>PC</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionNDS">
   <property name="icon">
    <iconset>
     <normaloff>icons/nds.png</normaloff>icons/nds.png</iconset>
   </property>
   <property name="text">
    <string>NDS</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionGBA">
   <property name="icon">
    <iconset>
     <normaloff>icons/gba.png</normaloff>icons/gba.png</iconset>
   </property>
   <property name="text">
    <string>GBA</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionSNES">
   <property name="icon">
    <iconset>
     <normaloff>icons/snes.png</normaloff>icons/snes.png</iconset>
   </property>
   <property name="text">
    <string>SNES</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionNES">
   <property name="icon">
    <iconset>
     <normaloff>icons/nes.png</normaloff>icons/nes.png</iconset>
   </property>
   <property name="text">
    <string>NES</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionWII">
   <property name="icon">
    <iconset>
     <normaloff>icons/wii.png</normaloff>icons/wii.png</iconset>
   </property>
   <property name="text">
    <string>WII</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionGC">
   <property name="icon">
    <iconset>
     <normaloff>icons/gc.png</normaloff>icons/gc.png</iconset>
   </property>
   <property name="text">
    <string>GC</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionGENESIS">
   <property name="icon">
    <iconset>
     <normaloff>icons/genesis.png</normaloff>icons/genesis.png</iconset>
   </property>
   <property name="text">
    <string>GENESIS</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset>
     <normaloff>icons/preferences.svg</normaloff>icons/preferences.svg</iconset>
   </property>
   <property name="text">
    <string>Preferences</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionTop">
   <property name="icon">
    <iconset>
     <normaloff>icons/top.svg</normaloff>icons/top.svg</iconset>
   </property>
   <property name="text">
    <string>Top</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionBottom">
   <property name="icon">
    <iconset>
     <normaloff>icons/bottom.svg</normaloff>icons/bottom.svg</iconset>
   </property>
   <property name="text">
    <string>Bottom</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset>
     <normaloff>icons/new.svg</normaloff>icons/new.svg</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionRemove">
   <property name="icon">
    <iconset>
     <normaloff>icons/remove.svg</normaloff>icons/remove.svg</iconset>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionImport">
   <property name="icon">
    <iconset>
     <normaloff>icons/import.svg</normaloff>icons/import.svg</iconset>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionExport">
   <property name="icon">
    <iconset>
     <normaloff>icons/export.svg</normaloff>icons/export.svg</iconset>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset>
     <normaloff>icons/info.svg</normaloff>icons/info.svg</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionSync">
   <property name="icon">
    <iconset>
     <normaloff>icons/sync.svg</normaloff>icons/sync.svg</iconset>
   </property>
   <property name="text">
    <string>Sync</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
